### ISSUE TITLE
help file check

### ISSUE THREAD

**Issue by TomasTomecek** [user-cont/colin#83]
* [ ] colin should check whether an image, or container has a help file
  * [ ] the name can be `help.1` or `README.md`
  * [ ] in the past it also could have been `help.md`

https://fedoraproject.org/wiki/Container:Guidelines#Help_File

**Comment by lachmanfrantisek** [comment]
It's already there..;-)

We found (with @phracek ), that:
- `redhat` speaks only about `help.1`
- `fedora` about `help.1` or `README.md`.

For now, the checks cover:
- redhat `help.1`
- fedora `help.1` or `README.md`

What about `help.md`? Is it another option or some deprecation?

**Comment by TomasTomecek** [comment]
cool, no work!

I actually think that it was renamed from help.md to README.md; let me check.

**Comment by TomasTomecek** [comment]
I agree with you that help.md is undefined but people still use it. So maybe do a check and flag it as deprecated?
